src/Entity/Speaker.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ */
+class Speaker
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private ?int $id = null;
+
+    /**
+     * @ORM\OneToOne(targetEntity="App\Entity\User", cascade={"persist", "remove"})
+     */
+    private ?User $user = null;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $firstname;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $lastname;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private ?string $twitter;
```

Comment:
In der Talk Entity habe ich beispielsweise für Github-Repos oder Slides Attributes hinterlegt, mit denen wir dann auch dynamisch neue Felder erstellen können. Wäre das hier dann nicht auch sinnvoll?

Reply:
Hab ich übersehen, sorry. Ja, können wir natürlich auch so machen.

Reply:
Hab ich übersehen, weil war noch nicht drin... kann ich natürlich so einbauen ^^

Reply:
Was war noch nicht drin?

Reply:
Talk hat keine Relation zu Attribute ^^

Reply:
Ahh ups :D